pyanaconda/installclasses/fedora_silverblue.py

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (C) 2017  Red Hat, Inc.  All rights reserved.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from pyanaconda.installclasses.fedora_workstation import FedoraWorkstationInstallClass
+from pyanaconda.product import productVariant
+
+__all__ = ["FedoraSilverBlueInstallClass"]
+
+
+class FedoraSilverBlueInstallClass(FedoraWorkstationInstallClass):
+    name = "Fedora SilverBlue"
+    stylesheet = "/usr/share/anaconda/pixmaps/workstation/fedora-workstation.css"
+
+    if productVariant != "SilverBlue":
```

Comment:
The name seems to be `Silverblue`.

Reply:
Where did you find that?

Reply:
Everywhere: https://github.com/rhinstaller/anaconda/issues/1505, https://pagure.io/teamsilverblue/issue/20, https://teamsilverblue.org/, https://mojefedora.cz/silverblue-atomic-workstation-pod-novou-znackou/

Reply:
Heh, OK :)